Cognite.Simulator.Extensions/SimulatorsExtensions.cs

```diff
@@ -0,0 +1,71 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using CogniteSdk.Alpha;
+using Cognite.Extractor.Common;
+using CogniteSdk;
+using CogniteSdk.Resources.Alpha;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Cognite.Simulator.Extensions
+{
+    /// <summary>
+    /// Class containing extensions to the CDF Simulators resource with utility methods
+    /// </summary>
+    public static class SimulatorsExtensions {
+        private static ILogger _logger = new NullLogger<Client>();
+
+        /// <summary>
+        /// Updates the logs in chunks for a simulator resource.
+        /// </summary>
+        /// <param name="cdfSimulators">The SimulatorsResource instance.</param>
+        /// <param name="id">The ID of the simulator.</param>
+        /// <param name="items">The list of log data entries.</param>
+        /// <returns>The updated SimulatorsResource instance.</returns>
+        public static async Task<SimulatorsResource> UpdateLogsBatch(
```

Comment:
```suggestion
        public static async Task UpdateLogsBatch(
```
we don't return anything